### PR TITLE
We worked on master branch.

### PR DESCRIPTION
**Pull request by milosvukadinovic**

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -12,32 +12,24 @@ def edit
```
Comment (rvvergara) [inline app/controllers/users_controller.rb]: @milosvukadinovic, 

In relation to your question:

> We haven't added name and bio to sign up process. Wouldn't the validation interfere with that?

Please look at `devise` 's documentation on how you can add these fields so you can use them in your sign up process.

File: app/controllers/friendships_controller.rb [hunk 2]
```diff
@@ -24,9 +24,11 @@ def create
   end
```
Comment (rvvergara) [inline app/controllers/friendships_controller.rb]: @milosvukadinovic, in relation to your reply

> Friendship.sender would be invalid as it is a list of users. Probably no method error? :D Working on fixing it.

This [line of code](https://github.com/azdrenymeri/social-network/blob/af3e97a0711932a01e73bb6aa129a6b1c580ace3/app/controllers/friendships_controller.rb#L7) indeed resulted in an error (#14). I see you guys haven't fixed it yet. But you have already pinpointed the problem. That's a good start

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (azdrenymeri) [reply app/controllers/users_controller.rb]: I have already done that but my coding partner maybe forgot to check the signup view

File: app/controllers/friendships_controller.rb (hunk 2 above)
Comment (azdrenymeri) [reply app/controllers/friendships_controller.rb]: It is fixed now

**Pull request merged**